community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/DynamicProcessorAssigner.java

```diff
@@ -51,7 +51,7 @@
 
     public DynamicProcessorAssigner( Configuration config )
     {
-        super( 100, MILLISECONDS );
+        super( 1, SECONDS );
```

Comment:
magic?

Reply:
As magic as 100 ms. It was just checked unnecessarily often.